index.html

```diff
@@ -4300,28 +4624,60 @@ <h3 id="Warrior">Warrior</h3>
             <tr>
                <td>Core
                </td>
-               <td>Tough
+               <td>Ignore Pain
                </td>
-               <td><strong>You are harder to kill than the others.</strong>
+               <td><strong>There is no need for evasion or armour when you can shrug off
+                     pain without issue. </strong>
                   <p>
-                     Gain an additional 2 HP per level in Warrior.
+                     You may use your STR bonus to calculate your AC as if it were DEX.
+                     You also ignore the effects of difficult terrain.
                </td>
                <td>Self
                </td>
                <td>Pass
                </td>
             </tr>
+            <tr>
+               <td>Core
```

Comment:
The core abilities are really cool and thematic, and set the tone of the warrior well. 

It brings back my fear of everyone getting loads of the core abilities though.   Maybe bring back the the discussion on "focus"es where you can choose core's once every long rest?

Reply:
I think we need them tbh, with only 3 ability uses for the actual abilities you get, we're going to end up with people just moving and attacking quite a bit at low levels. Unless we can offer some interesting way for players to regain ability uses aside from just resting